src/locales/fr.json

```
@@ -1,11 +1,12 @@
 {
-    "unlock": {
-        "title": "Authentification",
-        "subtitle": "Déverouiller votre cozy pour continuer",
-        "label": "Mot de passe",
-        "show": "afficher",
-        "hide": "masquer",
-        "abort": "Quitter",
-        "unlock": "Dévérouiller"
-    }
-}
\ No newline at end of file
+  "unlock": {
+    "title": "Authentification",
+    "subtitle": "Déverouiller votre cozy pour continuer",
+    "error": "Mot de passe incorrect, essayer à nouveau",
```

```suggestion
    "error": "Votre mot de passe semble incorrect, veuillez essayer à nouveau",
```

I think we should strive for conversational text.

---

FWIW I tend to agree, but the "dry" version was chose on purpose here, so I'll leave it in.

---

What is the purpose ?

---

in this case, `s/essayer/essayez/`

---

The text is shorter so the UI looks less heavy.

I asked about the `essayer` vs `essayez` as well, and the infinitive is purposeful. FWIW Googlealso tends to use the infinitive form in messages like that.